flink-runtime/src/main/java/org/apache/flink/streaming/api/operators/InternalTimeServiceManager.java

```diff
@@ -111,6 +112,7 @@ interface Provider extends Serializable {
         <K> InternalTimeServiceManager<K> create(
                 TaskIOMetricGroup taskIOMetricGroup,
                 CheckpointableKeyedStateBackend<K> keyedStatedBackend,
+                AsyncKeyedStateBackend<K> asyncKeyedStateBackend,
```

Comment:
How about merge the two parameter into one state backend? Like this
```suggestion
                PriorityQueueSetFactory<K> factory,
                KeyGroupRange keyGroupRange,
```